scripts/leapfrog/playbooks/logstash-start.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: Start logstash serivce
+  hosts: logstash_all
+  user: root
+  tasks:
```

Comment:
Should this be some form a rolling restart or can we really start all logstash services at once?

Reply:
So you want to add more script to examine if all logstash containers service are back online?

Reply:
cool, I'll add it